pages/dkp/kommander/2.1/release-notes/index.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: layout.pug
```

Comment:
Something's off here with this file. We should make sure we remove this from the PR before we merge it.